clvm-traits/src/wrappers.rs

```diff
@@ -0,0 +1,19 @@
+use crate::{ClvmDecoder, ClvmEncoder, FromClvm, FromClvmError, ToClvm, ToClvmError};
+
+/// A wrapper for an intermediate CLVM value. This is required to
+/// implement `ToClvm` and `FromClvm` for `N`, since the compiler
+/// cannot guarantee that the generic `N` type doesn't already
+/// implement these traits.
+pub struct Raw<N>(pub N);
```

Comment:
so, in order to convert to- and from `NodePtr`, we have to wrap in in a `Raw<NodePtr>()` now?

Reply:
Only when lazy-evaluating within generic ToClvm and FromClvm implementations. NodePtr explicitly implements ToClvm/FromClvm so you can rely on that for code using this library, but there can't be a blanket implementation for any type N.

Reply:
So unless you're working with generic N node types, you can keep using NodePtr or whatever specific type you were before